regalia-logansquare/build.gradle

```diff
@@ -25,18 +27,16 @@ android {
 }
 
 dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
 
-    api "com.bluelinelabs:logansquare:1.3.7"
-
-    compileOnly "io.realm:realm-android-library:$realmVersion"
-
+    api 'com.bluelinelabs:logansquare:1.3.7'
+    api 'com.fasterxml.jackson.core:jackson-core:2.10.2'
+    api 'com.fasterxml.jackson.module:jackson-module-kotlin:2.10.2'
     compileOnly project(':regalia')
-
-    testImplementation "io.realm:realm-android-library:$realmVersion"
-
     testImplementation project(':test')
-
     testAnnotationProcessor 'com.bluelinelabs:logansquare-compiler:1.3.7'
```

Comment:
This probably needs to be `testKapt` if it is actually used, but I don't think it is needed at all.